llvm/include/llvm/IR/RepoTicket.h

```diff
@@ -59,46 +58,55 @@ struct GONumber {
 
 /// A structure of a global object (GO) information.
 struct GOInfo {
-  /// GO's initial hash value which does not include the hash of its dependents.
+  /// The Initial-digest is the hash of an object (a function or global
+  /// variable) without consideration for any references to other objects.
   DigestType InitialDigest;
-  /// A set of global objects which the GO's hash depenendent on.
-  DependenciesType Dependencies;
-  /// A set of global variables which the GO's hash contributed to.
-  ContributionsType Contributions;
+  /// The Contributions of an object X are those objects Y which transitively
+  /// reference X and where a potential optimisation to either X or any of Y may
+  /// invalidate both.
+  SmallGOVec Contributions;
+  /// The Dependencies of an object X are those objects Y which X transitively
+  /// references and which are not Contributions.
```

Comment:
I'm not sure using X and Y helps the description here. What about:
"The Dependencies of an object are those objects which it transitively references but are not "Contributions"

Reply:
Fixed, thanks.